app/assets/javascripts/application-secret.js

```diff
@@ -0,0 +1,23 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or vendor/assets/javascripts of plugins, if any, can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// the compiled file.
+//
+// WARNING: THE FIRST BLANK LINE MARKS THE END OF WHAT'S TO BE PROCESSED, ANY BLANK LINE SHOULD
+// GO AFTER THE REQUIRES BELOW.
+//
+/* eslint-disable spaced-comment */
+//= require jquery
+//= require jquery-ui
+//= require jquery_ujs
+//= require action_cable
+//= require_directory ./lib
+//= require ./cloudcarousel-secret
+//= require ./metamaps.secret.bundle
+//= require ./Metamaps.ServerData
+//= require homepageVimeoFallback
+/* eslint-enable spaced-comment */
```

Comment:
"Why are you using multiple bundles exactly? I'm not opposed, just interested"

Because we've been ripping and tearing at all the J's code on the other branch to make this special mode work, without trying to keep the code workable for the main mode at the same time. Once we are on to the path we really like we do it better, and make the two modes able to co-exist in the same bundle/codebase